required/latex-lab/latex-lab-footnotes.dtx

```diff
@@ -1191,6 +1206,7 @@
     \floatingpenalty \@MM
     \hsize\columnwidth
     \@parboxrestore
+    \UseTaggingSocket{para/restore}
```

Comment:
wouldn't it be better (?) to add this socket into \@parboxrestore ? because that is the 2e hook to do such things so the socket would show up in most places that needs it automatically.

Reply:
Probably.  But I didn't dare to make such a drastic change that can affect many places without more tests.